config/environments/development.rb

```diff
@@ -37,11 +37,6 @@
 
   config.action_controller.forgery_protection_origin_check = ENV['DISABLE_FORGERY_REQUEST_PROTECTION'].nil?
 
-  ActiveSupport::Logger.new($stdout).tap do |logger|
```

Comment:
This is now built-in?

Reply:
...I forget the exact changes in each of 7.2 vs 8.0, but yes I think something like this has been made a default config. Production still has the more explicit assignment so that it includes the configured log_formatter.